docs/topics/coroutines-basics.md

```diff
@@ -245,35 +245,39 @@ Done
 
 <!--- TEST -->
 
-## Coroutines ARE light-weight
+## Coroutines are light-weight
 
-Run the following code:
+Coroutines are less resource-intensive than JVM threads. Code that exhausts the
+JVM's available memory when using threads can be expressed using coroutines
+without hitting resource limits. For example, the following code launches
+100000 distinct coroutines that each wait 5 seconds and then print a period
+('.'):
 
 ```kotlin
 import kotlinx.coroutines.*
 
-//sampleStart
 fun main() = runBlocking {
+//sampleStart
     repeat(100_000) { // launch a lot of coroutines
         launch {
             delay(5000L)
             print(".")
         }
     }
-}
 //sampleEnd
+}
 ```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
```

Comment:
This would go against our narrative :)
Try running https://play.kotlinlang.org/#eyJ2ZXJzaW9uIjoiMS42LjEwIiwiY29kZSI6ImltcG9ydCBrb3RsaW54LmNvcm91dGluZXMuKlxuLy9zYW1wbGVTdGFydFxuZnVuIG1haW4oKSA9IHJ1bkJsb2NraW5nIHtcbi8vc2FtcGxlU3RhcnRcbiAgICByZXBlYXQoMTAwXzAwMCkgeyAvLyBsYXVuY2ggYSBsb3Qgb2YgY29yb3V0aW5lc1xuICAgICAgICBsYXVuY2gge1xuICAgICAgICAgICAgZGVsYXkoNTAwMEwpXG4gICAgICAgICAgICBwcmludChcIi5cIilcbiAgICAgICAgfVxuICAgIH1cbn1cbi8vc2FtcGxlRW5kIiwicGxhdGZvcm0iOiJqYXZhIiwiYXJncyI6IiJ9
However lightweight coroutines are, they still take *some* memory, more that our playground allocates.

Reply:
ah, I figured something like this might be the case--I added a comment noting as much so that future contributors don't make the same mistake, I felt there was probably a good reason for this one not being runnable but didn't see any immediate indication as to why, thus the PR--thanks for letting me know!